src/app/dashboard/accounts/[guid]/page.tsx

```diff
@@ -122,13 +122,9 @@ export default function AccountPage({ params }: AccountPageProps): JSX.Element {
   });
 
   const split1 = new Split();
-  split1.value = 0;
-  split1.quantity = 0;
   split1.fk_account = account;
 
   const split2 = new Split();
-  split2.value = 0;
-  split2.quantity = 0;
```

Comment:
This fixes the "bug" in the UI where we had to type twice for entering a quantity/value!